contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
Is it possible to also include this file somewhere in the pack (e.g. `contrib/examples/actions/flows/print_text.sl`) so this action is self sustaning?